### PR TITLE
restructure multileadtype and remove big test file

### PR DESCRIPTION
**Pull request by ElineOppersma**

### CODE REVIEW

**Comment by rspwarnaar** [comment]
def working_pipeline_exp(our_chosen_file): in helper_functions_legacy is the only function that is not moved yet. See the this file with my changes where I commented out all methods that are moved to their new locations.
When this function is moved, we can consider also deleting the legacy file as it contains only redundant code

**Review by rspwarnaar — CHANGES_REQUESTED** [review]
See helper_functions_legacy

**Comment by ElineOppersma** [comment]
Last function also moved from the legacy helper_functions and removed the legacy file. 
Note: notebooks should start now with:
`sys.path.insert(0, '..')`
whereafter the required modules can be imported, for exmaple:
`from resurfemg.preprocessing import filtering`

File: resurfemg/helper_functions_legacy.py [hunk 1]
```diff
@@ -1,1902 +0,0 @@
...
-    #  pick components with more peak
-    emg = pick_more_peaks_array(components)
-    # now process it in final steps
-    abs_values = abs(emg)
-    final_envelope_d = emg_highpass_butter(abs_values, 150, 2048)
-    final_envelope_a = naive_rolling_rms(final_envelope_d, 300)
-
-    return final_envelope_a
-
-
```
Comment (rspwarnaar) [inline resurfemg/helper_functions_legacy.py:609]: This function (working_pipeline_pre_ml) is not yet introduced in another submodule

**Review by rspwarnaar — CHANGES_REQUESTED** [review]
working_pipeline_pre_ml Moet nog naar pipelines, dan is ie goed :)

**Review by rspwarnaar — APPROVED** [review]

**Pull request merged**